core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/InstanceStatus.java

```diff
@@ -10,6 +10,7 @@ public enum InstanceStatus {
     SERVICES_UNHEALTHY,
     WAITING_FOR_REPAIR,
     STOPPED,
+    PENDING,
```

Comment:
Is this the property exposed to end users? Do we want to expose a PENDING state here? (Couldn't really find where this translation happens either. (Found InstanceSyncState where some other translations happen though)

Reply:
We can do one thing currently as we do not have a pending state so in case of upscale if a node on aws end is stuck on pending state than on CB end it is shown as stopped. So, that node will definitely be picked up while we try to upscale and than we can filter this node out in AwsInstanceConnector and force stop it and remove its Vm from the list of other VMs which would be send to poller. 
So, by this the full flow will be maintained as well.